src/coreclr/jit/gentree.cpp

```diff
@@ -16201,7 +16201,9 @@ GenTree* Compiler::gtNewRefCOMfield(GenTree*                objPtr,
             // helper needs pointer to struct, not struct itself
             if (pFieldInfo->helper == CORINFO_HELP_SETFIELDSTRUCT)
             {
-                assg = impGetStructAddr(assg, CHECK_SPILL_ALL, true);
+                // TODO: verify if flags matter here
```

Comment:
Can you please open an issue about this as well and tag all of these with `TODO-Bug?`?

I see some places here are definitely bugs, esp. w.r.t. missed volatiles.

Reply:
I'll open one after the PR gets merged.